src/modules/exec.ts

```diff
@@ -19,7 +19,8 @@ import { ExecSpec, ProgramSpec } from './component';
 import { ProjectCache } from './project-cache';
 import { ProjectConfig } from './project-config';
 
-const CACHE_OUTPUT_REGEX: RegExp = /(\w+)\s*=\s*(\'.*?\'|\".*?\"|\w+)\s+\>\>\s+VELOCITAS_CACHE/;
+const CACHE_QOUTED_OUTPUT_REGEX: RegExp = /(\w+)\s*=\s*(\'.*?\'|\".*?\"|\w+)\s+\>\>\s+VELOCITAS_CACHE/;
+const CACHE_ARRAY_OUTPUT_REGEX: RegExp = /(\w+)\s*=\s*(\[\s*(?:\"([^\"]+)(?:\"\s*,\s*|\"))*\])\s+\>\>\s+VELOCITAS_CACHE/;
```

Comment:
Do we really need multiple regexes which operate on the same input? I would recommend we only run one regex against the line which returns `key=value`, then we parse value whether it starts and ends with `[` and `]` and then process its values as an array, otherwise we treat it is a literal of either string or number.